docs/how-to/jupyter-notebook.rst

```diff
@@ -83,6 +83,52 @@ Confirm the GPU is detected and usable by running:
 
    tf.config.list_physical_devices('GPU')
 
+Create an Intel GPU-enabled notebook
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create an Intel GPU-enabled Jupyter Notebook with IPEX or ITEX.
```

Comment:
Can we add a link under "IPEX" and "ITEX" for referencing the user to further info?

Reply:
posted the question on the MM channel, not sure which links we can use here